court_ball_game/tests.py

```diff
@@ -0,0 +1,64 @@
+from .models import CourtBallGame
+from court.models import Court
+from player.models import BallGame
+from decimal import Decimal
+from django.db.utils import IntegrityError
+from django.core.exceptions import ValidationError
+import pytest
+
+
+@pytest.mark.django_db
+class TestCourtBallGameModel:
+
+    @pytest.fixture
+    def court(self):
+        court = Court.objects.create(x=Decimal('11'), y=Decimal('22'),
+                                     city='TEST_CITY', neighborhood='TEST_NEIGHBORHOOD', max_players=30)
+        return court
+
+    def test_verified_instance(self, court):
+        court_in_db = Court.objects.get(pk=court.pk)
+        assert court_in_db == court
+
+    def test_create_court_ball_game(self, court):
+        test_court = court
+        court_ball_game = CourtBallGame.objects.create(court_id=test_court, ball_game=BallGame.Basketball)
+        assert isinstance(court_ball_game, CourtBallGame)
+
+    def test_create_court_ball_game_invalid_court_id(self):
+        with pytest.raises(IntegrityError):
+            CourtBallGame.objects.create(court_id=None, ball_game=BallGame.Basketball)
+
+    def test_create_court_ball_game_invalid_ball_game(self, court):
+        test_court = court
+        with pytest.raises(ValidationError):
+            CourtBallGame.objects.create(court_id=test_court, ball_game='Invalid-Ball-Game').full_clean()
+
+    def test_update_court_ball_game(self, court):
+        test_court = court
+        court_ball_game = CourtBallGame.objects.create(court_id=test_court, ball_game=BallGame.Basketball)
+        court_ball_game.ball_game = BallGame.Volleyball
+        court_ball_game.save()
+        updated_court_ball_game = CourtBallGame.objects.get(id=court_ball_game.id)
+        assert updated_court_ball_game.ball_game == BallGame.Volleyball
+
+    def test_delete_court_ball_game(self, court):
+        test_court = court
+        court_ball_game = CourtBallGame.objects.create(court_id=test_court, ball_game=BallGame.Basketball)
+        court_ball_game.delete()
+        with pytest.raises(CourtBallGame.DoesNotExist):
+            CourtBallGame.objects.get(id=court_ball_game.id)
```

Comment:
```suggestion
        id = court.id
        court_ball_game = CourtBallGame.objects.create(court_id=court, ball_game=BallGame.Basketball)
        court_ball_game.delete()
        with pytest.raises(CourtBallGame.DoesNotExist):
            CourtBallGame.objects.get(id=id)
```

Reply:
Changed to another version, 
Please tell me what do you think about it.